docs/src/main/paradox/amqp.md

```diff
@@ -54,6 +56,33 @@ Scala
 Java
 : @@snip [snip](/amqp/src/test/java/docs/javadsl/AmqpDocsTest.java) { #create-sink }
 
+### With flow
```

Comment:
Put the flow variant first in the docs. Sinks are not as important when flows are available.

Reply:
Done.